hof.settings.json

```diff
@@ -0,0 +1,13 @@
+{
+  "appName": "UKVI Ending of Tenancy",
+  "theme": "govUK",
+  "behaviours": [
+    "hof/components/clear-session"
+  ],
+  "routes": [
+    "./apps/end-tenancy"
+  ],
+  "session": {
+    "name": "ukvi-end-tenancy.hof.sid"
```

Comment:
On the /cookies page the session ID is listed as hod.sid - one will need to be updated

Reply:
cookies page is updated with this now on the latest HOF 👍